.kres.yaml

```diff
@@ -261,20 +261,12 @@ spec:
     - name: grype-scan
       buildxOptions:
         enabled: true
-      depends:
-        - default
```

Comment:
should we keep the dependency here more to "serialize" the steps, so that we don't do buildx build connurrently?

Reply:
I can keep it, if that helps performance